matter_server/server/ota/__init__.py

```diff
@@ -0,0 +1,55 @@
+"""OTA implementation for the Matter Server."""
+
+from matter_server.server.ota import dcl
+
+HARDCODED_UPDATES: dict[tuple[int, int], dict] = {
+    # OTA requestor example app, useful for testing
+    (0xFFF1, 0x8001): {
+        "vid": 0xFFF1,
+        "pid": 0x8001,
+        "softwareVersion": 2,
+        "softwareVersionString": "2.0",
+        "cdVersionNumber": 1,
+        "softwareVersionValid": True,
+        "otaChecksum": "7qcyvg2kPmKZaDLIk8C7Vyteqf4DI73x0tFZkmPALCo=",
+        "otaChecksumType": 1,
+        "minApplicableSoftwareVersion": 1,
+        "maxApplicableSoftwareVersion": 1,
+        "otaUrl": "https://github.com/agners/matter-linux-example-apps/releases/download/v1.3.0.0/chip-ota-requestor-app-x86-64.ota",
```

Comment:
want to move this to the libs repo as well to prevent that you need to maintain this on your account ?

Reply:
The examples are also used by the developer add-on so I'd have to change things there too.

We can also delete it here, it is only really useful for testing during development.

Reply:
Whatever you prefer, we can always change it later.